FWCore/PythonParameterSet/src/MakePyBind11ParameterSets.cc

```diff
@@ -24,17 +24,17 @@ static void makePSetsFromString(std::string const& module) {
 namespace edm {
   namespace cmspybind11 {
     std::unique_ptr<ParameterSet> readConfig(std::string const& config) {
-      PyBind11ProcessDesc pythonProcessDesc(config);
+      PyBind11ProcessDesc pythonProcessDesc(config, true);
       return pythonProcessDesc.parameterSet();
     }
 
-    std::unique_ptr<ParameterSet> readConfig(std::string const& config, int argc, char* argv[]) {
-      PyBind11ProcessDesc pythonProcessDesc(config, argc, argv);
+    std::unique_ptr<ParameterSet> readConfig(std::string const& config, const std::vector<std::string>& args) {
+      PyBind11ProcessDesc pythonProcessDesc(config, true, args);
       return pythonProcessDesc.parameterSet();
     }
 
     void makeParameterSets(std::string const& configtext, std::unique_ptr<ParameterSet>& main) {
```

Comment:
Also a note for subsequent work (for core), this function should really return the `unique_ptr` as the `readConfig` does. We could also look into just returning the `ParameterSet` by value.

Reply:
Indeed, this interface seems a bit inconsistent and redundant in places, but I think the cleanup can happen later (since it has been that way for many years).